src/Platform/Microsoft.Testing.Platform/OutputDevice/Console/LoggerOutcome.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.OutputDevice.Console;
+
+/// <summary>
+/// Outcome of a test.
+/// </summary>
+internal enum LoggerOutcome
```

Comment:
Given what we are doing with it, I think we need only the 3 states (passed, failed, skipped).